.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint and Test
+
+on:
```

Comment:
Hi @BRIHMAT-Naoui , I would suggest to test it in another branch before applying it on the main directly, maybe a `dev` branch, what do you think about that ?

Reply:
yes ala you are right